dv/uvm/core_ibex/waivers/coverage_waivers_xlm.tcl

```diff
@@ -0,0 +1,25 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Exclude standard primitives that are verified elsewhere
+exclude -type "prim_secded_*"
+exclude -type "prim_lfsr"
+exclude -type "prim_flop"
+exclude -type "prim_prince"
+exclude -type "prim_subst_perm"
+exclude -type "prim_ram_1p*"
```

Comment:
@weicaiyang can you take a look at these whether they look good?
I believe we have only have testbenches for `prim_lfsr` and `prim_prince`, but not for the other primitives - see here: https://reports.opentitan.org/hw/top_earlgrey/dv/summary/latest/report.html

Reply:
@GregAC do we need to exclude all of them? we typically only exclude the ones that are pre-verifed, and collect coverage on the others...

Reply:
Agree with @msfschaffner. We can only exclude pre-verified IPs, otherwise, we need to have a good reason on the exclusion.

For the fair amount of uncovered, is it due to unreachable code? We may consider running UNR to take them out.

Reply:
Thankfully it appears we still hit our V2 coverage goals when we only exclude `prim_prince` and `prim_lfsr`. So I'll leave it like that for now. We can get a UNR flow setup for Ibex post V2.

Reply:
Ok this sounds great, thanks for checking @GregAC!